tests/test_materialized_view_helpers.py

```diff
@@ -0,0 +1,142 @@
+import logging, logging.config
+
+import pytest
+from pytest import fixture
+
+from sqlalchemy import Table, Column, Integer, BigInteger, Float, String, Date, DateTime, Boolean, ForeignKey, MetaData, Numeric, Interval
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.sql import select, text, literal_column
+from sqlalchemy.sql import column
+
+from pycds.util import generic_sesh
+from pycds.materialized_view_helpers import MaterializedViewMixin
+
+# Define some simple objects (and their tables) to test view helpers against
+
+Base = declarative_base()
+metadata = Base.metadata
+
+
+class Thing(Base):
+    __tablename__ = 'things'
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    description_id = Column(Integer, ForeignKey('descriptions.id'))
+
+    def __repr__(self):
+        return '<Thing(id=%d, name=%s, desc_id=%d)>' % (self.id, self.name, self.description_id)
+
+
+class Description(Base):
+    __tablename__ = 'descriptions'
+    id = Column(Integer, primary_key=True)
+    desc = Column(String)
+
+    def __repr__(self):
+        return '<Description(id=%d, desc=%s)>' % (self.id, self.desc)
+
+
+class SimpleThing(Base, MaterializedViewMixin):
+    # __selectable__ = select([Thing]).where(Thing.id <= literal_column('3'))  # deeelightful
+
+    # less delightful but applicable to cases where we use text queries for selectable:
+    __selectable__ = text("""
+        SELECT *
+        FROM things
+        WHERE things.id <= 3
+    """).columns(Thing.id, Thing.name, Thing.description_id)
+
+    def __repr__(self):
+        return '<SimpleThing(id=%d, desc=%s)>' % (self.id, self.name)
+
+
+class ThingWithDescription(Base, MaterializedViewMixin):
+    # this works:
```

Comment:
This was really just notes to myself as I was figuring out how to name columns in a pure text selectable. I was planning on removing these, but obviously PR fever got the better of me.